python/ccf/clients.py

```diff
@@ -572,28 +563,24 @@ def __init__(
         host: str,
         port: int,
         ca: str,
-        cert: Optional[str] = None,
-        key: Optional[str] = None,
+        session_auth: Optional[Identity] = None,
+        signing_auth: Optional[Identity] = None,
         connection_timeout: int = DEFAULT_CONNECTION_TIMEOUT_SEC,
         description: Optional[str] = None,
         ws: bool = False,
-        disable_client_auth: bool = False,
     ):
         self.connection_timeout = connection_timeout
         self.description = description
         self.name = f"[{host}:{port}]"
         self.is_connected = False
+        self.sign = bool(signing_auth)
```

Comment:
For my own understanding - this is equivalent to `signing_auth is not None`?

Reply:
No, that's sign is falsy, so False, "", [], {} etc would also yield False. Anyway, this is removed now.